Project/Source/Components/ComponentTrail.cpp

```diff
@@ -101,6 +97,21 @@ void ComponentTrail::OnEditorUpdate() {
 		DeleteQuads();
 	}
 
+	if (ImGui::DragScalar("Texture Repeats", ImGuiDataType_U32, &nTextures)) {
+		if (nTextures <= 1) nTextures = 1;
+		if (nTextures > 50) nTextures = 50;
+		DeleteQuads();
+		EditTextureCoords();
+	}
+	ImGui::Checkbox("Color Over Trail", &colorOverTrail);
+	if (colorOverTrail) {
+		ImGui::DragFloat("Color Speed ", &colorSpeed, App->editor->dragSpeed2f, 0, 1);
+		ImGui::ColorEdit4("Init Color", initC.ptr(), ImGuiColorEditFlags_NoInputs);
+		ImGui::ColorEdit4("Medium Color", mediumC.ptr(), ImGuiColorEditFlags_NoInputs);
+		ImGui::ColorEdit4("Final Color", finalC.ptr(), ImGuiColorEditFlags_NoInputs);
+		if (ImGui::Button("Reset Color")) ResetColor();
```

Comment:
I think that is better to restart the color when the trail has finished the movement, aka when there isn't any quad drawn, instead of having a button